sentry_sdk/tracing.py

```diff
@@ -199,9 +200,11 @@ def start_child(self, **kwargs):
             **kwargs
         )
 
-        child._span_recorder = recorder = self._span_recorder
-        if recorder:
-            recorder.add(child)
+        span_recorder = (
```

Comment:
Does this mean the entire `Span._span_recorder` field should be moved to Transaction?

Reply:
In the long run, yes, I think so.